src/routing/mod.rs

```diff
@@ -411,6 +411,7 @@ async fn handle_connection_events(
                 stage.send_event(event).await;
             }
             ConnectionEvent::Disconnected(addr) => {
+                info!("Lost connection to {:?}", addr);
```

Comment:
This is probably not important enough to use `info`, is it? I'd use `debug` or even `trace` to avoid spamming the logs.

Reply:
right, I will change it to trace

Reply:
Nice catch @maqi  that kind of .is_ok() is possible and hides errors, so nice one.